docs/source/users/kernel-envs.md

```diff
@@ -55,6 +55,9 @@ There are several supported `KERNEL_` variables that the Enterprise Gateway serv
     expected to exceed that of the EG_KERNEL_LAUNCH_TIMEOUT set when Enterprise
     Gateway starts.
 
+  KERNEL_INFO_TIMEOUT=<from user> or EG_KERNEL_INFO_TIMEOUT=60
+    The time (in seconds) Enterprise Gateway will wait before giving up on a kernel
```

Comment:
Perhaps use the same text as the `EG_` variable (once clarified by previous comment), but also please add _how_ this is different (e.g., this only applies to the single kernel launch because its conveyed from the client or something).